### ISSUE TITLE
Why?

### ISSUE THREAD

**Issue by qutran** [himynameisdave/svb#24]
What a difference between this package and sveltejs/template except the moment that this packages makes not possible to customize you bundle configuration?

**Comment by himynameisdave** [comment]
Hey @qutran, great questions and thanks for asking! I will do my best to answer them here.

Basically, this project aims to take all the pain out of configuring a Svelte project. Some folks like being able to configure everything themselves, and that's great. But for myself (and many others) who just want to quickly get started with building Svelte apps, `svb` will make it easier for them to do so. It is heavily inspired by [`create-react-app`](https://github.com/facebook/create-react-app), which is amazing because users don't need to tinker with webpack or linter config.

### How is this different from the Svelte template?

You're correct, Svelte [already has a good template](https://github.com/sveltejs/template) for starting projects. However, one thing I find annoying is needing to clone/copy that repo. I am also unsure of how maintained and up-to-date that template is/will be going forward.

I personally believe the best configuration to get started on a project is zero configuration, ie: I don't need to have a `rollup.config.js` file and all its `devDependencies` installed at all. I can then also setup the best sensible defaults for the project.

### This package makes it impossible to configure your project.

At this point in `svb`s development, this is true, as this is the whole point of this package. However there are plans to have some sort of [eject command](https://github.com/himynameisdave/svb/issues/18) much like in `create-react-app`, so that after starting to build out a Svelte app if you wanted to control the config, you could do so at that point.

---

**tldr;** It's a trade-off. Zero-config means you don't need to think about configuration stuff and can just focus on building your app. 

I hope that this has helped address your questions!
